mirrord/cli/src/config.rs

```diff
@@ -24,6 +24,10 @@ pub(super) enum Commands {
     /// Operator commands eg. setup
     #[command(hide = true)]
     Operator(Box<OperatorArgs>),
+
+    /// List targets/resources like pods/namespaces in json format
+    #[command(hide = true)]
+    Ls(Box<LsArgs>),
```

Comment:
Use `rename` so internally in code it can be called `ListTargets` or something better.